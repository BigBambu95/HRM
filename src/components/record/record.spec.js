import React from 'react'
import Record from './record'

describe('Record', () => {
	it('Render record', () => {
		const record = shallow(
			<Record label='Профессия' field='Front-end developer' />
		)

		expect(record).toMatchSnapshot()
	})
})
