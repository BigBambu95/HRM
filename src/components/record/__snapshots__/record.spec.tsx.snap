// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Record Render record 1`] = `
<div
  className="record"
>
  <span>
    Профессия
  </span>
  <span>
    Front-end developer
  </span>
</div>
`;
